src/site/content/en/blog/pwas-on-oculus-2/index.md

```diff
@@ -514,6 +515,46 @@ from the Immersive Web Working Group's [WebXR Samples](https://immersive-web.git
 
 {% Video autoplay=true, muted=true, loop=true, playsinline=true, src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/nDm0rY5DvtyTbDgUqFTS.mp4" %}
 
+### Augmented/Mixed Reality (Passthrough)
+
+As announced at Meta Connect 2022, Meta Quest Browser
+[has added support](https://developer.oculus.com/documentation/web/webxr-mixed-reality/) for
+[WebXR Augmented Reality (AR)](/web-ar/), also known as Mixed Reality (MR), on Meta Quest 2 and Meta
+Quest Pro devices.
+
+Let's check a [slightly modified](https://a-frame-hello-world-ar.glitch.me) A-Frame
+[starter example](https://aframe.io/docs/1.3.0/introduction/#getting-started) with scaled-down
+models and hidden sky and plane for augmented reality.
+
+[A-Frame](https://aframe.io) is an open source web framework for building 3D/VR/AR experiences
+entirely out of declarative, reusable [custom HTML elements](/custom-elements-v1/) that are easy to
+read, understand, and copy-and-paste.
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://aframe.io/releases/1.3.0/aframe.min.js"></script>
+  </head>
+  <body>
+    <a-scene>
+      <a-entity scale="0.2 0.2 0.2">
+        <a-box position="-1 0.5 -3" rotation="0 45 0" color="#4CC3D9"></a-box>
+        <a-sphere position="0 1.25 -5" radius="1.25" color="#EF2D5E"></a-sphere>
+        <a-cylinder position="1 0.75 -3" radius="0.5" height="1.5" color="#FFC65D"></a-cylinder>
+        <a-plane position="0 0 -4" rotation="-90 0 0" width="4" height="4" color="#7BC8A4" hide-on-enter-ar></a-plane>
+        <a-sky color="#ECECEC" hide-on-enter-ar></a-sky>
+      </a-entity>
+    </a-scene>
+  </body>
+</html>
+```
+
+// TODO: add video
```

Comment:
```suggestion
// TODO: add video
```

Should we add the above demo as a Glitch?

Reply:
What do you mean? It's already here: https://github.com/GoogleChrome/web.dev/pull/9125/files#diff-c6a7fb61774789f401f0c1ce34ce6702e4e2e3e718f1ef862a2e45d881f80147R525

Reply:
You can embed Glitches: https://web.dev/handbook/web-dev-components/#embed-a-glitch.

Reply:
Good idea!

Reply:
@tomayac Done. :wink: